## File "./flocq/Calc/Bracket.v", line 654, characters 0-27:
## Error: The variable Z_div_mod_eq was not found in the current environment.

Index: flocq/Calc/Bracket.v
--- flocq/Calc/Bracket.v.orig
+++ flocq/Calc/Bracket.v
@@ -651,7 +651,7 @@ now apply Zpower_gt_1.
 now apply Z_mod_lt.
 rewrite <- 2!Rmult_plus_distr_r, <- 2!plus_IZR.
 rewrite Zmult_comm, Zplus_assoc.
-now rewrite <- Z_div_mod_eq.
+now rewrite <- Z_div_mod_eq_full.
 Qed.
 
 Theorem inbetween_float_new_location_single :
